src/utils/getS3SignedUrl.js

```diff
@@ -0,0 +1,26 @@
+const AWS = require('aws-sdk');
+const config = require('../config');
+
+const getS3SignedUrl = (operation, params) => {
+  const s3 = new AWS.S3({
+    apiVersion: '2006-03-01',
+    signatureVersion: 'v4',
+    region: config.awsRegion,
+  });
```

Comment:
These were necessary for s3 upload, are they necessary for download too?

Reply:
No, this is not necessary for download as bucket name is globally unique. I added this because I was using the same function to get the URL for upload and download. I'll split it up